exoctk/phase_constraint_overlap/phase_constraint_overlap.py

```diff
@@ -67,14 +77,36 @@ def calculate_obsDur(transitDur):
         Returns
         -------
         obsdur : float
-            The duration of the observation in hours. '''
+            The duration of the observation in hours. Maximum of 6 hours. '''
 
     obsDur = np.min((6, 3*transitDur+1))
 
     return obsDur
 
 
-def phase_overlap_constraint(target_name, period=None, t0=None, obs_duration=None, window_size=None):
+def get_transit_details(target_name):
```

Comment:
We have this portion of source code taken care of now. Feel free to remove this function.